tools/test-case-generators/format-request-map.json

```diff
@@ -814,6 +829,28 @@
       }
     }
   },
+  "iot-simplified-installer": {
+    "compose-request": {
+      "distro": "",
+      "arch": "",
+      "image-type": "iot-simplified-installer",
+      "repositories": [],
+      "filename": "installer.iso",
+      "ostree": {
+        "ref": "test/fedora/iot",
+        "url": "http://iot.example.com/repo"
+      },
+      "blueprint": {
+        "customizations": {
+          "installation_device": "/dev/vda",
+          "fdo": {
+            "manufacturing_server_url": "https;//fdo.example.com",
```

Comment:
I don't think that this might break anything, but you have a `;` instead of a `:` in this URL

Reply:
Oops ^_^

No, it wont break anything since those are just sample manifests that can't be built anyway (and of course the URL is never used).  I'll fix it if any other changes are required as well.